module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,48 @@
+#include "compression.hpp"
+#include <algorithm>
+#include <iostream>
```

Comment:
I think that newline between header and standard library includes would be nice.